### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

**Review by aina21 — APPROVED** [review]

File: javascript/javascript2/week2/script1.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    .filter(movie => movie.year >= 1980)
+    .filter(movie => movie.year <= 1989)
+console.log(`${oldMovies.length} Movies were made between 1980-1989 (including both the years)`);
+
+//4.Create a new array that has an extra key called tag. 
+//The tag is based on the rating: Good (>= 7), Average (>= 4 and < 7), Bad (< 4)
+
+const moviesWithTag = [];
+const goodMovie = movies.filter(movie => movie.rating >= 7);
+goodMovie.forEach(movie => movie.tag = "Good");
```
Comment (andrii1) [inline javascript/javascript2/week2/script1.js:31]: I think you can do const goodMovie = movies.filter(movie => movie.rating >= 7).forEach(movie => movie.tag = "Good"); with a dot.

File: javascript/javascript2/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+const myProducts = document.querySelector("ul");
+myProducts.setAttribute("class", "my-products");
+
+console.log(searchType);
+function renderProducts(products) {
+
+    for (let i = 0; i < products.length; i++) {
+        let myProductList = document.createElement("li");
+        myProductList.setAttribute("class", "my-products-list")
+        myProductList.style.listStyle = "none";
```
Comment (andrii1) [inline javascript/javascript2/week2/main.js:21]: why not to do it in css file? Just a question

File: javascript/javascript2/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        myProducts.innerHTML = "";
+        alert("This is not a Price")
+    } else {
+        myProducts.innerHTML = "";
+        renderProducts(cheapRander)
+    }
+});
+
+expensiveBtn.addEventListener("click", function () {
+    const inputPrice2 = +expensiveProducts.value;
```
Comment (andrii1) [inline javascript/javascript2/week2/main.js:76]: a better name for const? e.g. inputPriceExpensiveProducts

File: javascript/javascript2/week2/main.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        myProducts.innerHTML = "";
+        renderProducts(rander);
+    } else {
+        displayOutput.innerHTML = "No more products";
+    }
+});
+
+cheapBtn.addEventListener("click", function () {
+    const inputPrice = cheapProducts.value;
+    const cheapRander = products.filter(list => list.price < inputPrice).sort((a, b) => a.price - b.price);
```
Comment (andrii1) [inline javascript/javascript2/week2/main.js:65]: you mean Render?

**Pull request merged**